book-domain-service/app/routes.py

```diff
@@ -5,10 +5,10 @@
 @app.route('/<int:id>/checkout', methods=['PUT'])
 def checkout(id):
 
-    bookTitle = Book.query.get(id).title
+    book = Book.query.get(id)
 
-    if(bookTitle):
-        return Response("You checked out book id: " + str(id) + " name: " +
-        str(bookTitle), 200)
+    if(book):
```

Comment:
You don't need the parens for if statements. They are typically used to make complications conditions more readable. Here are a few examples:
```
# Single condition
if book:
    pass

# Multiple conditions
if book is not None and foo == 'bar':
    pass

# Complex conditions
if (book is not None and foo == 'bar') or (x == y):
    pass
```